src/Uno.Extensions.Navigation.UI/RouteResolver.cs

```diff
@@ -18,6 +21,8 @@ IViewRegistry views
 	protected RouteResolver(ILogger logger, IRouteRegistry routes, IViewRegistry views)
 	{
 		Logger = logger;
+		RouteMapss = routes;
```

Comment:
Should be `RouteMaps` instead of `RouteMapss` here also right ?

Reply:
Corrected